build-config/resources/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+distribution = "2201.8.0"
+org = "ballerinax"
+name = "googleapis.drive"
+version = "@toml.version@"
+authors = ["Ballerina"]
+repository = "https://github.com/ballerina-platform/module-ballerinax-googleapis.drive"
+keywords = ["Content & Files/File Management & Storage", "Cost/Free", "Vendor/Google"]
```

Comment:
I don't think these keywords are good. We have to decide a good set of keywords for the connectors. @shafreenAnfar @daneshk @NipunaRanasinghe WDYT?